server/admin_management/api/app/services/utils_service.py

```diff
@@ -0,0 +1,35 @@
+import os
+import logging
+
+from api.app.constants import AppEnv, ApiInstanceEnv, AwsTargetEnv
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_aws_target_env() -> AwsTargetEnv:
+    # TARGET_ENV is assigned from gov's AWS platform, does not exist in local (None).
+    return os.environ.get("TARGET_ENV")
```

Comment:
Hi Ian, I just realized this needs to be upper case "TARGET_ENV" instead of "target_env". Terraform has it in upper case.

Reply:
Yes, I think I noticed the other day but forgot again to mention... thanks!